.gitignore

```diff
@@ -30,6 +30,11 @@ build/
 # spa_c_files builds
 pvlib/spa_c_files/build*
 
+# spa_c_files source
+pvlib/spa_c_files/spa.c
+pvlib/spa_c_files/spa.h
+pvlib/spa_c_files/spa_tester.c
+
```

Comment:
Hm, on second thought I already removed the file and line 10 of ``.gitignore`` is ``*.so``. You may be seeing the file because of some git oddity when switching between branches.